dmsgpty/pty_gateway_windows.go

```diff
@@ -0,0 +1,33 @@
+//+build windows
+
+package dmsgpty
+
+import (
+	"golang.org/x/sys/windows"
+)
+
+// PtyGateway represents a pty gateway, hosted by the pty.SessionServer
+type PtyGateway interface {
```

Comment:
Can these type declarations be moved into a some common file for all builds?

Reply:
yup that's what I planned to, and I did. But `gob` needs all types to be registered, so I think it could be better if we just remove the interface (there's only one implementation anyway) and stick to just concrete type, what do you think?

Reply:
Ah, I missed that they are different. 
Well, interface might help with testing, other than that I don't see much reason in keeping it. It is just considered 'better code', because of better decoupling.

I'm not sure if we need it, but in case we do, I probably would try to make gateway take universally available data. Like instead of taking windows.Coord it would take some Coord type defined in our code. Windows implementation will then deal with converting generic Coord type into windows.Coord and back.